config/packages/doctrine.yaml

```diff
@@ -27,9 +27,9 @@ doctrine:
 #                dir: '%kernel.project_dir%/src/Entity'
 #                prefix: 'App\Entity'
 #                alias: App
-            Domain:
-                is_bundle: false
-                type: yml
-                dir: '%kernel.project_dir%/config/mapping/orm/query'
-                prefix: 'App\Domain'
-                alias: Domain
\ No newline at end of file
+#            Domain:
+#                is_bundle: false
+#                type: yml
+#                dir: '%kernel.project_dir%/config/mapping/orm/query'
```

Comment:
This dir is missing in the repo, I've commented everything to bypass this config